docs/installation/installation-using-docker-linux.md

```diff
@@ -1,5 +1,7 @@
 # Installation Using Docker for Linux
 
+**This guide is for version which is not released yet. See the [version for `v7.0.0-beta5`](https://github.com/shopsys/shopsys/blob/v7.0.0-beta5/docs/installation/installation-using-docker-linux.md).**
```

Comment:
`for version` or `for the version` ?

Reply:
Right, missing article :slightly_smiling_face: